privacyidea/static/components/user/views/user.add.html

```diff
@@ -1,5 +1,15 @@
+<style type="text/css">
+  .css-form input.ng-invalid.ng-touched {
+    background-color: #FA787E;
+  }
+
+  .css-form input.ng-valid.ng-touched {
+    background-color: #78FA89;
+  }
+</style>
+
 <div class="panel" ng-controller="userAddController">
-    <form name="formAddUser" role="form" validate>
+    <form name="User" role="form" novalidate>
```

Comment:
On a quick skim read of the code as I've not touched it for a while: I guess it was done as the controller getUserAttributes sets a more granular set of validations. Again I guess that my lack of experience with Angular may be showing again :(